jetpack_container.sh

```diff
@@ -3,9 +3,9 @@
 ansible_dir=/jetpack
 
 # build jetpack container
-podman build -t jetpack .
+sudo podman build -t jetpack .
 
 # run jetpack container
-podman run -it \
+sudo podman run --privileged -it \
```

Comment:
Why does podman need to run in privileged mode?

Reply:
Maybe try something like this instead?

`podman run --cap-add=sys_admin,mknod --device=/dev/fuse --security-opt label=disable quay.io/podman/stable podman run ubi8-minimal echo hello`

Source : https://www.redhat.com/sysadmin/podman-inside-container

Reply:
> Why does podman need to run in privileged mode?

--privileged is needed to run podman inside a container

Reply:
> Maybe try something like this instead?
> 
> `podman run --cap-add=sys_admin,mknod --device=/dev/fuse --security-opt label=disable quay.io/podman/stable podman run ubi8-minimal echo hello`
> 
> Source : https://www.redhat.com/sysadmin/podman-inside-container

this might work without --privileged. will give it a try

Reply:
> Maybe try something like this instead?
> 
> `podman run --cap-add=sys_admin,mknod --device=/dev/fuse --security-opt label=disable quay.io/podman/stable podman run ubi8-minimal echo hello`
> 
> Source : https://www.redhat.com/sysadmin/podman-inside-container

I did hit some errors with this change

Reply:
Then I think it might be okay to use --privileged.

Reply:
Add a comment in README.md saying that jetpack container will use `--privileged` flag.

Reply:
> Add a comment in README.md saying that jetpack container will use `--privileged` flag.

ack